torch/serialization.py

```diff
@@ -103,6 +103,29 @@ def storage_to_tensor_type(storage):
 def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL):
     """Saves an object to a disk file.
 
+    There are two main approaches for serializing and restoring a model.
```

Comment:
model -> neural network model

Also, this blob has to be moved to notes, "Serialization Best Practices", and a reference to the notes can be inserted here in the reference docs (reference docs are not full notes).